cmd/broker/ingress/main.go

```diff
@@ -52,6 +51,14 @@ var (
 	kubeconfig = flag.String("kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 )
 
+// TODO make these constants configurable (either as env variables, config map, or part of broker spec).
```

Comment:
GH issue for this ?

Reply:
There is an old one on adding config-maps for timeouts. I think we can put all that in there... 
The issue is: https://github.com/knative/eventing/issues/1777